bin/pip.py

```diff
@@ -17,24 +17,29 @@
         remove      removed an installed package
         update      update an installed package
 """
-import sys
-import os
+from __future__ import print_function
+
 import ast
-import shutil
-import types
 import contextlib
-import requests
-import re
 import operator
-
-import six
+import os
+import re
+import shutil
+import sys
+import types
 from distutils.util import convert_path
 from fnmatch import fnmatchcase
+
+import requests
+import six
 # noinspection PyUnresolvedReferences
-from six.moves import filterfalse
+from six.moves.configparser import NoSectionError, SafeConfigParser
 
 from stashutils.extensions import create_command
 
+# noinspection PyUnresolvedReferences
+
+
```

Comment:
This `noinspection` comment looks like it doesn't do anything anymore. Since the import it belonged to was merged into another one with the same `noinspection` comment, I think this one can safely be removed.